packages/mask/src/plugins/WalletService/services/wallet/password.ts

```diff
@@ -48,9 +48,9 @@ export async function changePassword(newPassword: string) {
 
 export function validatePassword(unverifiedPassword: string) {
     if (!unverifiedPassword) return false
-    if (unverifiedPassword.length < 8) return false
+    if (unverifiedPassword.length < 6) return false
     if (unverifiedPassword.length > 20) return false
-    return [/[A-Z]/, /[a-z]/, /\d/, /[^\dA-Za-z]/].filter((x) => x.test(unverifiedPassword)).length >= 2
```

Comment:
We should consult the PM before removing it.

Reply:
consulted